src/main/java/it/gov/pagopa/pu/bff/connector/process_executions/client/IngestionFlowFileSearchClient.java

```diff
@@ -25,9 +26,10 @@ public PagedModelIngestionFlowFile getIngestionFlowFiles(IngestionFlowFileFilter
       return processExecutionsApisHolder.getIngestionFlowFileSearchControllerApi(accessToken)
         .crudIngestionFlowFilesFindByOrganizationIDFlowTypeCreateDate(
           String.valueOf(ingestionFlowFileFilters.getOrganizationId()),
-          ingestionFlowFileFilters.getFlowFileType().toString(),
-          ingestionFlowFileFilters.getCreationDateFrom(),
-          ingestionFlowFileFilters.getCreationDateTo(),
+          ingestionFlowFileFilters.getFlowFileType().stream().map(
+            FlowFileTypeEnum::toString).toList(),
+          ingestionFlowFileFilters.getCreationDateFrom()!=null?ingestionFlowFileFilters.getCreationDateFrom().toLocalDateTime():null,
+          ingestionFlowFileFilters.getCreationDateTo()!=null?ingestionFlowFileFilters.getCreationDateTo().toLocalDateTime():null,
```

Comment:
Before to convert into LocalDateTime you should as first convert the OffsetDateTime to the current offset.
Let's move the logic into a common utility method
```suggestion
          ingestionFlowFileFilters.getCreationDateFrom()!=null?ingestionFlowFileFilters.getCreationDateFrom().toLocalDateTime():null,
          ingestionFlowFileFilters.getCreationDateTo()!=null?ingestionFlowFileFilters.getCreationDateTo().sameInstantAtZone(ZoneId.systemdefault()).toLocalDateTime():null,
```